src/chia_dialect.rs

```diff
@@ -186,10 +186,10 @@ impl Dialect for ChiaDialect {
         match ext {
             // The BLS extensions (and coinid) operators were brought into the
             // main operator set as part of the hard fork
-            0 => OperatorSet::BLS,
-            1 if (self.flags & ENABLE_KECCAK) != 0 => OperatorSet::Keccak,
+            0 | 1 => OperatorSet::Default,
```

Comment:
When I reviewed your first PR, I (for some reason) was thinking that the BLS softfork had id 0, so 1 was the next one. I realize I was wrong. Can you think of ways to prevent this mistake in the future?
There should probably be a comment here explaining what operator set 1 *is* (or was, prior to hard fork).
Maybe it even makes sense to preserve the `OperatorSet::BLS` enum value.